modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -11,7 +11,13 @@ Following this procedure reduces the total update duration and the number of tim
 
 .Prerequisites
 
-* Review the release notes for {product-title} <4.y+1> and <4.y+2>
+* Review the release notes for {product-title} <4.y+1> and <4.y+2>.
 * Review the release notes and product lifecycles for any layered products and Operator Lifecycle Manager (OLM) Operators. Some may require updates either before or during an EUS-to-EUS update.
 * Ensure that you are familiar with version-specific prerequisites, such as the removal of deprecated APIs, that are required prior to updating from {product-title} <4.y+1> to <4.y+2>.
-
+* If your cluster uses in-tree vSphere volumes, update vSphere to version 7.0u3L+ or 8.0u2+.
++
+[IMPORTANT]
+====
+If you do not update vSphere to 7.0u3L+ or 8.0u2+ before initiating an {product-title} update, known issues might occur with your cluster after the update.
```

Comment:
Do we want to use the title for the link text here: Known Issues with OpenShift 4.12 to 4.13 or 4.13 to 4.14 vSphere CSI Storage Migration?